src/icons/glyphs/svgs/IvermectinGlyph.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+function SvgIvermectinGlyph(props: React.SVGProps<SVGSVGElement>) {
+  return (
+    <svg
+      width={48}
+      height={48}
+      viewBox="0 0 48 48"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <g clipPath="url(#ivermectin-glyph_svg__clip0)" fill="currentColor">
```

Comment:
I _think_ you can remove these `clipPath` elements 🤔 The [Working with SVG's documentation](https://www.notion.so/curology/Working-with-SVGs-c70a743abbe144f1898e9c1095b5b267#ae5c0648f0a2440d8ae87eb4ff2f337c) recommends pruning superfluous SVG markup that Figma usually adds. Looks like some of the similar icons (hydrocortisone, hydroquinone) do not include these masks.